terraform/README.md

```diff
@@ -33,9 +33,11 @@ The below steps rely on you first configuring access to the Terraform state in s
     ```bash
     terraform init
     terraform plan
-    terraform apply
     ```
 
+1. GitHub Actions will apply the changes when your PR is merged. To force `terraform apply` to run even if you didn't
+update any terraform files, make any change to `terraform/<env>/.force-action-apply`
```

Comment:
Added this bit because `terraform apply` didn't run after the only change you had to make was updating the workflow file.